### ISSUE TITLE
Adding "add/delete/update experiment" functionality

### ISSUE THREAD

**Issue by yvikhlya** [GEOS-ESM/GEOS_Web#1]
The web site currently has basic functionality. This repo can be cloned to web server (polar or train) and tested. Example url is following:

https://gmao.gsfc.nasa.gov/intranet/research/modeling/agcm/geos5/yury/GEOS_Web/index.php

The web site uses the following lqlite3 database file with list of experiments:
```
/gmao/intranet/research/modeling/agcm/geos5/yury/experiments.db
``` 
which has to be located one level up from GEOS_Web clone path. File locations can be easily changed later when the web site is released. I chose sqlite3 as DBMS, because I don't need any help from sysadmins to set it up. An alternative would be to use MySQL server, but then sysadmins need to set it up properly. 

The next step is to add interface for manipulating experiments data base (add/delete/modify). The best way to do this, IMO, is through a web interface. In this case sysadmins should set up proper permissions for modeling group to access experiments database. The alternative would be to make users ssh to web server and run a python script, sort of what we do now, and I could code this without sysadmin's help. 

@mathomp4, @tclune are welcomed to discussion. What makes more sense to you: sqlite3 + python script or MySQL+web interface?

**Comment by mathomp4** [comment]
@yvikhlya I have to imagine a web interface would be "easier" for most users. No longer a need to log into train, etc. Just go to a web page and do it all.

But, as you say, a new web page means security fun. But I suppose if it is behind the GMAO Firewall, it might be simple-ish.

**Comment by yvikhlya** [comment]
@mathomp4 What I can do without any help right now is to change db file permission to `go+w` and try to do a web interface. The whole development portal is behind PIV login anyway. We can't do this for extranet pages thought, but we can make extranet only for reading.

Users will still need to use scp to copy plot files to web server.

**Comment by mathomp4** [comment]
> Users will still need to use scp to copy plot files to web server.

Ahh. I forgot about that. Well, I suppose for most users, `moveplot` (or whatever it is called) will do that automatically for them as the experiment runs, right?

**Comment by yvikhlya** [comment]
Partially resolved by #2 Still need a solution for database access permissions.
